xilem/src/any_view.rs

```diff
@@ -22,186 +21,50 @@ use crate::{MasonryView, MessageResult, ViewCx, ViewId};
 /// Note that `Option` can also be used for conditionally displaying
 /// views in a [`ViewSequence`](crate::ViewSequence).
 // TODO: Mention `Either` when we have implemented that?
-pub type BoxedMasonryView<State, Action = ()> = Box<dyn AnyMasonryView<State, Action>>;
-
-impl<State: 'static, Action: 'static> MasonryView<State, Action>
-    for BoxedMasonryView<State, Action>
-{
-    type Element = DynWidget;
-    type ViewState = AnyViewState;
+pub type AnyWidgetView<State, Action = ()> =
+    Box<dyn AnyView<State, Action, ViewCtx, Pod<DynWidget>> + Send + Sync>;
 
-    fn build(&self, cx: &mut ViewCx) -> (masonry::WidgetPod<Self::Element>, Self::ViewState) {
-        self.deref().dyn_build(cx)
+impl<W: Widget> SuperElement<Pod<W>> for Pod<Box<dyn Widget>> {
```

Comment:
This implementation is technically mostly used for sequences, not `any_view`

Maybe there's a better file for it?